Mathlib/Analysis/SpecialFunctions/Pow/NNReal.lean

```diff
@@ -734,6 +734,9 @@ theorem le_rpow_one_div_iff {x y : ℝ≥0∞} {z : ℝ} (hz : 0 < z) : x ≤ y
   rw [rpow_mul, ← one_div, @rpow_le_rpow_iff _ _ (1 / z) (by simp [hz])]
 #align ennreal.le_rpow_one_div_iff ENNReal.le_rpow_one_div_iff
 
+theorem rpow_inv_lt_iff {x y : ℝ≥0∞} {z : ℝ} (hz : 0 < z) : x ^ z⁻¹ < y ↔ x < y ^ z := by
+  simp only [← not_le, ← one_div, le_rpow_one_div_iff hz]
```

Comment:
Can you check:
* `lt_rpow_inv` already exists for `ENNReal`. If not, add it.
* the corresponding lemmas for `Real` and `NNReal` exist. If not, add them.

Reply:
* x ≤ y ^ z⁻¹ ↔ x ^ z ≤ y:
  - Real.le_rpow_inv_iff_of_pos
  - NNReal.le_rpow_one_div_iff
  - ENNReal.le_rpow_one_div_iff
* x ^ z⁻¹ ≤ y ↔ x ≤ y ^ z:
  - Real.rpow_inv_le_iff_of_pos
  - NNReal.rpow_one_div_le_iff
  - ENNReal.rpow_one_div_le_iff
* x < y ^ z⁻¹ ↔ x ^ z < y :
  - Real.lt_rpow_inv_iff_of_pos
  - NNReal <- missing
  - ENNReal.lt_rpow_one_div_iff
* x ^ z⁻¹ < y ↔ x < y ^ z:
  - Real.rpow_inv_lt_iff_of_pos
  - NNReal <- missing
  - ENNReal <- this PR

The NNReal and ENNReal modules use `1 / z` instead of `z⁻¹`. I will add the missing theorems, but I won't change the existing API, unless it is deemed the right time to do so.

Reply:
We are trying to phase out `1 / z`, so yes please add the `z^-1` versions and deprecate the `1 / z` ones